src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,19 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+	public static final String BLOCK_VALIDATION_REGEX = "\\d+";
+	public String value;
+	
+	public Block() {
```

Comment:
Java provides a default constructor anyway, so you don;t really need to do this.